### PR TITLE
Attach default props when spans are started

### PR DESCRIPTION
**Pull request by pbomb**
We see scenarios where spans are started, new action is recorded, then the span is ended. under these scenarios, the span contains information from the previous trace, causing data issues.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,9 +1,10 @@
 {
   "name": "rallymetrics",
-  "version": "3.1.0",
+  "version": "4.0.0",
   "description": "Metrics aggregation for Rally Software",
   "main": "builds/rallymetrics.js",
   "scripts": {
+    "build": "grunt build",
```
Comment (songawee) [inline package.json]: :+1:

File: src/aggregator.js [hunk 2]
```diff
@@ -243,23 +260,24 @@ Aggregator.prototype.startSpan = function(options) {
...
+            return;
+          }
           options = options || {};
           options.stop = aggregator.getRelativeTime(options.stopTime);
 
           if (aggregator._shouldRecordEvent(this.data, options)) {
               aggregator._finishEvent(this.data, _.extend({
                   status: 'Ready'
-              }, options));
+              }, _.omit(options, 'stopTime')));
```
Comment (songawee) [inline src/aggregator.js]: Is this omitted because we are using a different value for stopTime?

**Comment by songawee** [comment]
Seems to be ok. Just making sure we always use the current traceId. :ship:

**Comment by pbomb** [comment]
I removed traceId because I didn't see where we were using it.

File: src/aggregator.js (hunk 2 above)
Comment (pbomb) [inline src/aggregator.js]: yeah, see above on line 247: `options.stop` is what needs to be sent to the beacon. `stopTime` is also getting sent but is just noise.

**Pull request closed without merging**